### ISSUE TITLE
[WinUI 3 Gallery: Media-> Sound]: Inappropriate button name is announced by the narrator for the buttons present under "Play Specific System Sound" heading.

### ISSUE THREAD

**Issue by sabareesanms** [microsoft/WinUI-Gallery#1968]
**Environment Details:**

**OS Version:** Dev (OS Build 27881.1000)

**App:** WinUI 3 Gallery

**App Version:** 2.6.0.0 (SDK 1.7)

**AT:** Screen Reader

**Pre-Requisite:**

Turn on Narrator (Ctrl + Win + Enter).

**Repro Steps:**

**Actual Result:** UI 3 Gallery app. Navigate to search box using tab key. Type 'Sound' in search box and hit enter. Navigate to 'Sound' card and activate it. Navigate to the buttons present under "Play Specific System Sound" heading. Listen to narrator announcement.  Actual Result:

**Actual Result:**

**Actual Result:**

**Actual Result:**

**Actual Result:**

**Expected Result:** tton name is announced by the narrator for the buttons present under "Play Specific System Sound" heading. When user navigates to narrator is announcing as "Right pointing black triangle shape, Focus, button".  Expected Result: Proper name property should be defined for the buttons present under "Play Specific System Sound" heading. When user navigates to it narrator should announce as "Focus, button".  User Impact:

Inappropriate button name is announced by the narrator for the buttons present under "Play Specific System Sound" heading. When user navigates to narrator is announcing as "Right pointing black triangle shape, Focus, button".

**Expected Result:**

Proper name property should be defined for the buttons present under "Play Specific System Sound" heading. When user navigates to it narrator should announce as "Focus, button".

**User Impact:**

**User Impact:**

**MAS Reference:** ual impairments who rely on screen reader will be impacted if narrator announces inappropriate button name.  MAS Reference:

**MAS Reference:** ual impairments who rely on screen reader will be impacted if narrator announces inappropriate button name.  MAS Reference:

**MAS Reference:** ual impairments who rely on screen reader will be impacted if narrator announces inappropriate button name.  MAS Reference:

**MAS Reference:** ual impairments who rely on screen reader will be impacted if narrator announces inappropriate button name.  MAS Reference:

**MAS Reference:** ual impairments who rely on screen reader will be impacted if narrator announces inappropriate button name.  MAS Reference:

Users with visual impairments who rely on screen reader will be impacted if narrator announces inappropriate button name.

**MAS Reference:**

**MAS 4.2.1 – Object Information**

**MAS 4.2.1 – Object Information**

**MAS 4.2.1 – Object Information**

**Trap ID:** 1.6 - Poor Grouping

**Trap ID:** 1.6 - Poor Grouping

**Trap ID:** 1.6 - Poor Grouping

**Trap ID:** 1.6 - Poor Grouping

**Trap ID:** 1.6 - Poor Grouping

**Trap ID:**

1.6 - Poor Grouping

**“Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.”**
